climada/util/calibrate/base.py

```diff
@@ -158,6 +159,102 @@ def plot_impf_set(self, **plot_kwargs):
         """
         return self.impf_set.plot(**plot_kwargs)
 
+    def plot_impf_variability(
+        self,
+        cost_func_diff: float = 0.1,
+        p_space_df: Optional[pd.DataFrame] = None,
+        plot_haz: bool = True,
+        plot_impf_kws : dict = {},
+        plot_hist_kws : dict = {},
```

Comment:
Careful! No mutable default arguments!
```suggestion
        plot_impf_kws: Optional[dict] = None,
        plot_hist_kws: Optional[dict] = None,
```
Needs some more adjustments in the code then.